hoot-core/src/main/cpp/hoot/core/algorithms/linearreference/WaySublineMatchString.h

```diff
@@ -121,18 +121,7 @@ class WaySublineMatchString
    */
   bool touches(const WaySublineMatchString& other) const;
 
-  QString toString() const
-  {
-    QString result;
-    result += QString("matches:\n");
-    QStringList l;
-    for (size_t i = 0; i < _matches.size(); i++)
-    {
-      l.append(_matches[i].toString());
-    }
-    result += l.join("\n");
-    return result;
-  }
+  QString toString() const;
```

Comment:
oh, yeah you were staying good for doing it...was thinking I needed to make a change.  Yeah, I don't like it either when those header files end up with anything more than the slightest bit of code in them.